TauAnalysis/MCEmbeddingTools/plugins/CaloCleaner.cc

```diff
@@ -1,11 +1,11 @@
 #include "TauAnalysis/MCEmbeddingTools/plugins/CaloCleaner.h"
 
 #include "DataFormats/EcalRecHit/interface/EcalRecHit.h"
+#include "DataFormats/HcalRecHit/interface/CastorRecHit.h"
```

Comment:
Why do we need to bother with castor?

Reply:
That is of course true and most castor associated code can be removed without changing anything. But if I remove the [following line](https://github.com/KIT-CMS/cmssw/blob/embedding_update_for_run3/TauAnalysis/MCEmbeddingTools/python/customisers.py#L652):
```python
process.mix.digitizers.castor.doNoise = cms.bool(False)
```
I get different distributions of my hadronic tau and my muon kinematics.
So this line overwrites the default config from [here](https://github.com/cms-sw/cmssw/blob/master/SimGeneral/MixingModule/python/castorDigitizer_cfi.py).
And I also verified that it is not an issue of random numbers or the rng seed as the generator particles look exactly the same w/ or w/o that line.
Does anyone have a clue why this line changes our events and whether we need this line?
Maybe an expert like @civanch ?